R/cohenFleissKappa.R

```diff
@@ -0,0 +1,202 @@
+#
+# Copyright (C) 2021 University of Amsterdam
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+cohenFleissKappa <- function(jaspResults, dataset, options) {
+  
+  ready <- length(options[["variables"]]) > 1
+  
+  dataset <- .cohensFleissKappaReadData(dataset, options)
+    
+  if(options[["cohensKappa"]])
+    jaspResults[["cohensKappa"]] <- .cohensKappa(dataset, options, ready)
+  
+  if(options[["fleissKappa"]])
+    jaspResults[["fleissKappa"]] <- .fleissKappa(dataset, options, ready)
+  
+  return()
+}
+
+# Read in the dataset (copied from .reliabilityReadData)
+.cohensFleissKappaReadData <- function(dataset, options) {
+  variables <- unlist(options[["variables"]])
+  if (is.null(dataset)) {
+    dataset <- .readDataSetToEnd(columns.as.factor = variables)
+  }
+  return(dataset)
+}
+
+.cohensKappa <- function(dataset, options, ready){
```

Comment:
```suggestion
.cohensKappa <- function(dataset, options, ready) {
```
I think the convention is to always leave a space before a `{`

Reply:
Thanks, will keep it in mind.